keanu-project/src/test/java/io/improbable/keanu/templating/SequenceTest.java

```diff
@@ -17,7 +17,7 @@
     @Test
     public void youCanGetTheLastItem() {
         int numItems = 10;
-        Sequence sequence = new Sequence(numItems);
+        Sequence sequence = new Sequence(numItems, this.hashCode(), null);
```

Comment:
I think standard usage would be to use the builder - the user shouldn't really be constructing sequences manually like this. In this case it doesn't matter so much because it's only a test.

Reply:
Yep indeed.